dynamic_rest/serializers.py

```diff
@@ -414,12 +414,12 @@ def get_link_fields(self):
                 not (
                     # Skip sideloaded fields
                     name in self.fields and
-                    not field.serializer.id_only()
+                    isinstance(self.request_fields.get(name), dict)
```

Comment:
Can you add this logic into a helper like `self.is_field_sideloaded` in case we need to do this again for other meta functionality?